zgw/documenten-api/src/main/kotlin/com/ritense/documentenapi/client/ConfidentialityLevel.kt

```diff
@@ -1,18 +1,36 @@
 package com.ritense.documentenapi.client
 
-enum class ConfidentialityLevel(val key: String) {
-    OPENBAAR("openbaar"),
-    BEPERKT_OPENBAAR("beperkt_openbaar"),
-    INTERN("intern"),
-    ZAAKVERTROUWELIJK("zaakvertrouwelijk"),
-    VERTROUWELIJK("vertrouwelijk"),
-    CONFIDENTIEEL("confidentieel"),
-    GEHEIM("geheim"),
-    ZEER_GEHEIM("zeer_geheim");
+import com.fasterxml.jackson.annotation.JsonProperty
+
+enum class ConfidentialityLevel {
+    @JsonProperty("openbaar")
+    OPENBAAR,
```

Comment:
Not sure if `@JsonProperty` also works for serializing. Because this object is also used in the request body to create an informatieobject

Reply:
I added a test for it: ConfidentialityLevelTest